### DISCUSSION TITLE
Send message after starting bot

### DISCUSSION THREAD

**Discussion by oneafter909** [python-telegram-bot/python-telegram-bot#4063] [Q&A] [answered]
Hi, i'm switching the bot's API python-telegram-bot from 13.15 to 20.7. 
In 13.15 I use this code to send a message to me when the bot starts:

```
def main():
    "Main thread"
        updater = Updater(TOKEN, use_context=True)
        dp = updater.dispatcher
        dp.add_handler(CommandHandler("start", startCommand))
	...
	bot.send_message(owner, text=f"{VERSION} {BUILD}")
        updater.start_polling()
        updater.idle()
```
how can I do with the new 20.7? I've tried a bunch of codes but none worked.

**Comment by Poolitzer** [comment, marked as answer]
[application.post_init()](https://docs.python-telegram-bot.org/en/v20.7/telegram.ext.applicationbuilder.html#telegram.ext.ApplicationBuilder.post_init)

**Reply by oneafter909** [reply]
Thanks!
